sites/public/src/patterns/ExpandableSection.module.scss

```diff
@@ -0,0 +1,51 @@
+.expandable-section {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  .header {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--seeds-bg-color-surface-primary);
+
+    .header-content {
+      width: 100%;
+      display: flex;
+      color: var(--seeds-text-color);
+      padding: var(--seeds-spacer-content);
+      flex-grow: 1;
+      text-align: start;
+      .heading {
+        font-size: var(--seeds-type-heading-size-2xl);
+        font-family: var(--seeds-font-alt-sans);
+        margin-block-end: 0;
+      }
+    }
+
+    .button-container {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+
+  .header-button {
+    width: 100%;
+    height: 100%;
+    font-size: var(--seeds-s10);
+    line-height: normal;
+    font-weight: lighter;
+    padding-inline: var(--seeds-spacer-section);
+    display: flex;
+    align-items: center;
+    color: var(--seeds-color-primary);
+  }
+}
+
+.content {
+  margin-block-end: var(--seeds-spacer-section);
+  border-block-start: var(--seeds-border-1) solid var(--seeds-border-color);
+}
+
+.button-wrapper {
+  width: 100%;
+}
```

Comment:
question: Is our standard practice to have CSS classes specific to each component and not use them in other places? I noticed there was a class in a different module in this PR called `full-width-button` that was also just `width: 100%`. Do we not anticipate having a module with often used classes?

Reply:
The module files are by design specific to a page, as the class names are randomized, so we couldn't do a module but we could and will gather classes that tend to come up multiple times and put them in a global scss file. We do anticipate having that, but I think before we have more Seeds uptake, it won't be revealed what those repeated ones will be. I'd prefer I think to keep discovering those patterns as we uptake if that's okay!

Reply:
I'm also curious to understand if we'd want to add any to Seeds specifically, or maybe if classes like that come up a lot or don't, it actually indicates something we should add to a component.

Reply:
Gotcha! Yeah, I do like that having these separate scss files per module makes it much easier to identify where they are being used